src/controllers/csi/metadata/metadata.go

```diff
@@ -41,7 +41,7 @@ type Volume struct {
 
 // NewVolume returns a new Volume if all fields are set.
```

Comment:
comment should be updated: `version` can be empty when we create a volume that has no version yet because of failures